src/serverenvironment.cpp

```diff
@@ -1926,9 +1921,16 @@ u16 ServerEnvironment::addActiveObjectRaw(std::unique_ptr<ServerActiveObject> ob
 			errorstream<<"ServerEnvironment::addActiveObjectRaw(): "
 				<<"could not emerge block for storing id="<<object->getId()
 				<<" statically (pos="<<p<<")"<<std::endl;
```

Comment:
not directly related but it would be a good opportunity to clean the message a bit:
```suggestion
			errorstream << "ServerEnvironment::addActiveObjectRaw(): "
				<< "could not emerge block " << p << " for storing id="
				<< object->getId() << " statically" << std::endl;
```